src/components/Glp/SwapErrorModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+import { ethers } from "ethers";
+import { getNativeToken } from "../../config/Tokens";
+import { InfoTokens, Token } from "../../domain/tokens/types";
+import Modal from "../Modal/Modal";
+import { get1InchSwapUrl } from "../../domain/common";
+import { getLowestFeeTokenForBuyGlp } from "../../domain/tokens/utils";
+
+const { AddressZero } = ethers.constants;
+
+type Props = {
+  swapToken: Token;
+  isVisible: boolean;
+  setIsVisible: () => void;
+  chainId: number;
+  glpAmount: BigNumber;
+  usdgSupply: BigNumber;
+  totalTokenWeights: BigNumber;
+  glpPrice: BigNumber;
+  swapUsdMin: BigNumber;
+  infoTokens: InfoTokens;
+};
+
+interface LowestFeeToken extends Token {
```

Comment:
1. It's better to use this style:  `type LowestFeeToken = Token & {...additionalProps}` just for a more minimalistic and consistent codestyle
2. Looks like this type should be defined in /domain, somewhere near getLowestFeeTokenForBuyGlp function
3. I thinking, is it really necessary to define a new type of token and call it so specialized (LowestFeeToken)? 

There is a risk that this way we will have a lot of inherited token types with different field sets, which will confuse

It seems to me that here we can do something like this:

```
const {token: Token, fees: number, amountLeftToDeposit: BigNumber} = getLowestFeeTokenForBuyGlp(...)

```

Or

```
const {token: Token, feesSpects: {...}} = getLowestFeeTokenForBuyGlp(.)
```